docs/advanced/shebang.md

```diff
@@ -0,0 +1,109 @@
+---
+part: pixi/advanced
+title: Self-contained scripts
+description: Learn how to write self-contained shell scripts using `pixi exec`
+---
+
+# Using `pixi exec` to create self-contained scripts
+
+!!!warning "Only on Unix-like systems"
+    The following approach only works on Unix-like systems (i.e. Linux and macOS) since Windows does not support shebang lines.
+
+For simple scripts, you can use [`pixi exec`](../reference/cli.md#exec) to run them directly without needing to take care of installing dependencies or setting up a virtual environment.
+This can be done by adding a [shebang line](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the top of the script, which tells the system how to execute the script.
+Usually, a shebang line starts with `#!/usr/bin/env` followed by the name of the interpreter to use.
+
+Instead of adding an interpreter, you can also just add `pixi exec` at the beginning of the script.
+The only requirement for your script is that you must have `pixi` installed on your system.
+
+!!!tip "Making the script executable"
+    You might need to make the script executable by running `chmod +x my-script.sh`.
+
+```bash title="deploy-tf.sh"
+#!/usr/bin/env -S pixi exec --spec opentofu -- bash -e
+
+tofu init
+tofu plan -out=tfplan
+tofu apply tfplan
+tofu output
+```
+
+!!!info "Explanation what's happening"
+    The `#!` are magic characters that tell your system how to execute the script (take everything after the `#!` and append the file name).
+
+    The `/usr/bin/env` is used to find `pixi` in the system's `PATH`.
+    The `-S` option tells `/usr/bin/env` to use the first argument as the interpreter and the rest as arguments to the interpreter.
+    `pixi exec --spec opentofu` creates a temporary environment containing only `opentofu`.
+    `bash -e` (separated with `--`) is the command that is executed in this environment.
+    So in total, `pixi exec --spec opentofu -- bash -e deploy-tf.sh` is being executed when you run `./deploy-tf.sh`.
```

Comment:
I don't think the explanation should be in an admonition, as it's pretty important for this document.

Reply:
since it's important, i wanted to highlight it this way

<img width="808" alt="image" src="https://github.com/user-attachments/assets/551349b3-d4ac-4832-bc4d-2bdda796e400" />

---

if it wan't in a `info` block, it doesn't stand out as much

<img width="778" alt="image" src="https://github.com/user-attachments/assets/41bf4bd9-3936-4cbb-b044-39acfcbbca32" />

but if you feel strongly about this, we can also make it a regular text block

Reply:
I can see it now. I don't feel strongly.